website/docs/packages/reference/query-module.md

```diff
@@ -70,56 +82,70 @@ When enabled, this property displays a confirmation modal each time before refre
 
 ## JS module
 
-These properties are customizable options available within the JS editor of the JS module. They can be accessed by adding the JS module in the App.
+These properties are customizable options available within the JS editor of the JS module. They can be accessed by adding the JS module in the App. Additionally, you can add multiple JS modules from the same package, each with different configurations.
 
-Additionally, you can add multiple JS modules from the same package, each with different configurations.
 
+### Parameters 
 
-<ZoomImage
-  src="/img/js-module-setting.png" 
-  alt="JS modules image"
-  caption=""
-/>
-
-### Function setting
-
-This setting allows you to configure each function available in the JS module according to your requirements.
+This property displays all the parameters available for the specified function.
 
-#### Function name
 
-<dd>
+#### Inputs
 
-This setting displays all the function names available in the JS module. 
 
+The input property allows you to pass dynamic values from your app to the JS module. The input parameters are only available if they are defined in the JS function. You cannot edit the input name or JS module configuration from the App; you can only pass values to the available inputs. 
 
+:::info
+When passing data from the app to the JS module using both **Inputs** and during **runtime** (e.g., `login.run({ email: <email>, password: <password> })`), the runtime parameters will take precedence over the values set in the UI.
```

Comment:
The information shown in this box is accurate for query modules but for JS Modules the syntax is different. When you want to pass arguments in runtime to a JS Function (login) that exist in a JS Module (authenticationUtils) you need to use the following syntax:

`authenticationUtils.login('email@domain.com', 'dsoi3$dfssn)`


Let me know if you have questions about the difference in both. I can help on clarifying.